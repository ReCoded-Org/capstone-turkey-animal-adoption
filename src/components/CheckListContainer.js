import React from "react";
import CheckList from "./CheckList";
import { Container, Row } from "react-bootstrap";
import "./CheckList.css";


const ChecklistContainer = ({ data }) => {
    console.log(data)
    return (

        <Container>
            <Row className="py-5">
                {data.map((info) => <CheckList text={info.text} title={info.title} link={info.link} />)}
            </Row>
        </Container>
    )
}

export default ChecklistContainer;