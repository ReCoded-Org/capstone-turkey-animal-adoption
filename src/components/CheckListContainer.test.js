import React from "react";
import renderer from "react-test-renderer";
import ChecklistContainer from "./CheckListContainer";

const data = [ 
    {text:"hello",
    title:"yoyoyo",
    link:"denemeLink"
  }, {text:"hello",
  title:"yoyoyo",
  link:"ikiucdort"
  }, {text:"hello",
  title:"yoyoyo",
  link:"ikiucdort"
  }, {text:"hello",
  title:"yoyoyo",
  link:"ikiucdort"
  }
  ]

it("renders correctly", () => {
  const tree = renderer
    .create(
      <ChecklistContainer data={data} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});