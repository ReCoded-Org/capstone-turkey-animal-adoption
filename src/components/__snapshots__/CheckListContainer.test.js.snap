// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="py-5 row"
  >
    <div
      className="checkList col-lg-4"
    >
      <div
        className="checkListLogo"
      >
        <img
          className="catLogo"
          src="catLogo.png"
        />
      </div>
      <h2>
        yoyoyo
      </h2>
      <p
        className="checkListText"
      >
        hello
      </p>
      <a
        href="denemeLink"
      >
        <button
          className="checkListButton"
        >
          Read More
        </button>
      </a>
    </div>
    <div
      className="checkList col-lg-4"
    >
      <div
        className="checkListLogo"
      >
        <img
          className="catLogo"
          src="catLogo.png"
        />
      </div>
      <h2>
        yoyoyo
      </h2>
      <p
        className="checkListText"
      >
        hello
      </p>
      <a
        href="ikiucdort"
      >
        <button
          className="checkListButton"
        >
          Read More
        </button>
      </a>
    </div>
    <div
      className="checkList col-lg-4"
    >
      <div
        className="checkListLogo"
      >
        <img
          className="catLogo"
          src="catLogo.png"
        />
      </div>
      <h2>
        yoyoyo
      </h2>
      <p
        className="checkListText"
      >
        hello
      </p>
      <a
        href="ikiucdort"
      >
        <button
          className="checkListButton"
        >
          Read More
        </button>
      </a>
    </div>
    <div
      className="checkList col-lg-4"
    >
      <div
        className="checkListLogo"
      >
        <img
          className="catLogo"
          src="catLogo.png"
        />
      </div>
      <h2>
        yoyoyo
      </h2>
      <p
        className="checkListText"
      >
        hello
      </p>
      <a
        href="ikiucdort"
      >
        <button
          className="checkListButton"
        >
          Read More
        </button>
      </a>
    </div>
  </div>
</div>
`;
